cmd/component.go

```diff
@@ -248,6 +248,91 @@ var componentListCmd = &cobra.Command{
 	},
 }
 
+var componentUpdateCmd = &cobra.Command{
```

Comment:
this should be in `update.go` and named `updateCmd`

Reply:
@kadel Done and fixed